Description: use dhis.conf
 dhis2-instance-create generates dhis.conf file instead of hibernate.properties
Index: dhis2-tools/usr/bin/dhis2-instance-create
===================================================================
--- dhis2-tools.orig/usr/bin/dhis2-instance-create	2016-07-08 16:57:44.000000000 +0200
+++ dhis2-tools/usr/bin/dhis2-instance-create	2016-07-08 16:57:44.000000000 +0200
@@ -1,8 +1,8 @@
 #!/bin/bash
-#       ____  __  ______________ 
+#       ____  __  ______________
 #      / __ \/ / / /  _/ ___/__ \
 #     / / / / /_/ // / \__ \__/ /
-#    / /_/ / __  // / ___/ / __/ 
+#    / /_/ / __  // / ___/ / __/
 #   /_____/_/ /_/___//____/____/
 #
 #   Script to deploy a dhis2 instance
@@ -70,7 +70,7 @@
 }
 ###############################################################
 
-# main entry point 
+# main entry point
 echo checking options
 while getopts p:nh opt
 do
@@ -132,16 +132,16 @@
   sudo -u postgres psql -c  "GRANT ALL ON DATABASE $DBNAME TO $TARGET;"
 fi
 
-sudo -u $TARGET sh -c "cat > $DHIS2HOME/hibernate.properties" << EOT
-hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-hibernate.connection.driver_class = org.postgresql.Driver
-hibernate.connection.url = jdbc:postgresql://localhost:5432/$DBNAME
-hibernate.connection.username = $TARGET
-hibernate.connection.password = $PASSWORD
-hibernate.hbm2ddl.auto = update
+sudo -u $TARGET sh -c "cat > $DHIS2HOME/dhis.conf" << EOT
+connection.dialect = org.hibernate.dialect.PostgreSQLDialect
+connection.driver_class = org.postgresql.Driver
+connection.url = jdbc:postgresql://localhost:5432/$DBNAME
+connection.username = $TARGET
+connection.password = $PASSWORD
+connection.schema = update
 EOT
 
-sudo -u $TARGET chmod 0600  $DHIS2HOME/hibernate.properties
+sudo -u $TARGET chmod 0600  $DHIS2HOME/dhis.conf
 sudo -u $TARGET sed -i s/HTTPPORT/$HTTPPORT/ $DHIS2HOME/conf/server.xml
 
 sudo -u $TARGET sh -c "cat > $DHIS2HOME/logrotate.conf" << EOT
Index: dhis2-tools/usr/share/doc/dhis2-tools/copyright
===================================================================
--- dhis2-tools.orig/usr/share/doc/dhis2-tools/copyright	2016-07-08 16:57:44.000000000 +0200
+++ dhis2-tools/usr/share/doc/dhis2-tools/copyright	2016-07-08 16:57:44.000000000 +0200
@@ -1,135 +1,24 @@
-Format: http://svn.debian.org/wsvn/dep/web/deps/dep5.mdwn?op=file&rev=166
-Upstream-Name: Tomcat7
-Upstream-Contact: Merten Falk <nevesenin@cpan.org>
-Source: http://tomcat.apache.org/download-70.cgi
-Upstream-Authors:
-    Alex Chaffee
-    Alex Cruikshank [alex@epitonic.com]
-    Amy Roh
-    Andre de Jesus
-    Andrew R. Jaquith
-    Andy Clark
-    Aner Perez
-    Anil V (akv@eng.sun.com)
-    Anselm Baird-Smith
-    Arnaud  Le Hors, IBM
-    Bela Ban (modifications for synchronous replication)
-    Bill Barker
-    Bill Burke
-    Bip Thelin
-    Cedrik LIME
-    Carson McDonald
-    Costin@eng.sun.com
-    Craig R. McClanahan
-    Dan Milstein [danmil@shore.net]
-    Dan Sandberg
-    Daniel Rall
-    Danno Ferrin
-    David Becker
-    Denis Benoit
-    Eric Rescorla
-    Eric Ye, IBM
-    Fabien Carrion
-    Fabrizio Giustina
-    Filip Hanik
-    Gabriele Garuglieri
-    Gal Shachor [shachor@il.ibm.com]
-    Glenn L. Nielsen
-    Glenn Marcy, IBM
-    Greg Murray
-    Gunnar Rjnning
-    Hans Bergsten <hans@gefionsoftware.com>
-    Harish Prabandham
-    Henri Gomez [hgomez@apache.org]
-    Ignacio J. Ortega
-    Jacek Laskowski
-    Jacob Hookom [jacob@hookom.net]
-    James Duncan Davidson [duncan@eng.sun.com]
-    James Todd [gonzo@sun.com]
-    Jan Luehe
-    Jason Brittain
-    Jason Hunter [jch@eng.sun.com]
-    Jason van Zyl
-    Jayson Falkner
-    Jean-Francois Arcand
-    Jean-Frederic Clere
-    Jeffrey Rodriguez
-    John Holman
-    John McNally
-    Jon S. Stevens
-    Justyna Horwat
-    Keith Wannamaker [Keith@Wannamaker.org]
-    Kevin Seguin
-    Kief Morris (kief@kief.com)
-    Kin-man Chung
-    Larry Cable
-    Larry Isaacs
-    Malcolm Edgar
-    Mandar Raje
-    Mark Roth
-    Mark Thomas
-    Martin Cooper
-    Martin T Dengler [root@martindengler.com]
-    Mel Martinez [mmartinez@g1440.com]
-    Michael Glavassevich, IBM
-    Mladen Turk
-    Neil Graham, IBM
-    Nicola Ken Barozzi
-    Paul Speed
-    Peter Donald
-    Peter Lin
-    Peter Rossbach (pero@apache.org)
-    Pierre Delisle
-    Rafal Krzewski Rafal.Krzewski@e-point.pl
-    Rahul Srivastava, Sun Microsystems Inc.
-    Rainer Jung
-    Rajiv Mordani
-    Remy Maucharat
-    Remy Maucherat
-    Richard A. Sitze
-    Robert Field (inner SDEInstaller class)
-    Rod Waldhoff
-    Scott Sanders
-    Sean C. Sullivan
-    Sean Legassick
-    Sean Legassick
-    Shawn Bayern
-    Stan Bailes
-    Stefan Freyr Stefansson
-    Stefano Mazzocchi
-    TAMURA Kent, IBM
-    Takayuki Kaneko
-    Tim Fennell
-    Tim Funk
-    Tim Tye
-    Vivek Chopra
-    Yoav Shapira <yoavs@apache.org>
+Copyright (c) 2004-2007, University of Oslo
+All rights reserved.
 
-Files: *
-Copyright: 2000-2011 Apache Software Foundation.
-  2002, International Business Machines Corporation.
-License: Apache-2.0
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of the HISP project nor the names of its contributors may
+  be used to endorse or promote products derived from this software without
+  specific prior written permission.
 
-Files: debian/*
-Copyright:
-  2008,2011, Canonical Ltd.
-  2008, Paul Cager <paul-debian@home.paulcager.org>
-  2010, Thierry Carrez <thierry.carrez@ubuntu.com>
-  2011, Ernesto Hernanez-Novich <emhn@usb.ve>
-License: Apache-2.0
-
-License: Apache-2.0
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
- .
- http://www.apache.org/licenses/LICENSE-2.0
- .
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
- .
- On Debian GNU/Linux and Ubuntu systems, the complete text of the Apache 2.0 
- License can be found in the `/usr/share/common-licenses/Apache-2.0' file.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
